tests/chassis_modules_test.py

```diff
@@ -112,6 +112,43 @@ def test_config_incorrect_module(self):
         print(result.output)
         assert result.exit_code != 0
 
+    def test_midplane_show_all_count_lines(self):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["chassis-modules"].commands["midplane-status"], [])
+        print(result.output)
```

Comment:
Can we also add comparison with the actual show o/p for show chassis/midplane commands for comparison ? like as we do here for eg: (https://github.com/Azure/sonic-utilities/blob/master/tests/aclshow_test.py#L20)

Reply:
Done. Thanks!